### PR TITLE
docs: initial commit for the top-level README

### PR DESCRIPTION
**Pull request by rchincha**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+*                 @afflom @rchincha
```
Comment (afflom) [inline CODEOWNERS:4]: @joshgav, This is displaying an error: Unknown owner on line 4: make sure @rchincha exists and has write access to the repository. 

Might you please point us in the right direction to resolve this?

**Review by afflom — APPROVED** [review]
Thank you for seeding this content.

File: CODEOWNERS (hunk 1 above)
Comment (SteveLasker) [reply CODEOWNERS:4]: I believe this is due to the teams configuration not being updated, yet. Once this is merged, @joshgav can update the github team to reflect permissions and the error will go away.

File: README.md [hunk 2]
```diff
@@ -1,37 +1,132 @@
...
+for system packages like deb and rpm. This abundance of package types and
+formats adds complexity and risk for cloud application developers and
+architects trying to provision and run cloud-native workloads. Different tools
+and controllers must be carefully learned and installed to bundle, unbundle,
+verify and deploy each artifact type. This complexity slows product development
+by users, impedes innovation and collaboration amongst projects, and increases
+risks of insecure configuration. To reduce this complexity and facilitate
+collaboration and innovation, WG Artifacts will gather stakeholders from many
+CNCF and open source projects offering packaging, distribution and deployment
+mechanisms for bundles of configuration and code.
```
Comment (SteveLasker) [inline README.md:22]: Great leadup to the goal. Should we add another sentence that suggests our goal isn't to consolidate all the existing package managers, rather to continue investing in the existing, avoiding the need to create yet new package managers?

File: CODEOWNERS (hunk 1 above)
Comment (rchincha) [reply CODEOWNERS:4]: Accepted the invite, doesn't look to be an issue anymore.

File: README.md (hunk 2 above)
Comment (rchincha) [reply README.md:22]: Hopefully one doesn't need to add a new pkg mgr. But should the need arise, at least don't have to think about what kinds of metadata queries need to be supported. Documents coming out of this WG should provide said guidance. Furthermore, software coming out of the WG is general enough to be re-used/repurposed.

**Pull request merged**